packages/netlify-cms-core/src/backend.js

```diff
@@ -276,11 +276,8 @@ class Backend {
     const publishedEntry = await this.implementation
       .getEntry(collection, slug, path)
       .then(({ data }) => data)
-      .catch(error => {
-        if (error.status === 404 || error.message.includes(404)) {
-          return Promise.resolve(false);
-        }
-        return Promise.reject(error);
+      .catch(() => {
+        return Promise.resolve(false);
```

Comment:
This stops the error from making it back up the chain. How about making sure `error.message` is defined before using `includes`?

Reply:
We are not checking for the contents of the `error` argument anymore. For 99.9% of all cases, the error argument would mean `404` not found, so we just resolve here.

Reply:
Ah, didn't realize this was inside of `entryExist` - sounds good 👍